### ISSUE TITLE
dataset partition

### ISSUE THREAD

**Issue by YangYangTaoTao** [mattz10966/SPAPNet#3]
Have you considered that each person's data can theoretically only appear in either the training set or the test set?

**Comment by YangYangTaoTao** [comment]
You paper mentioned that, you cut a long video into short clips. I my experience,  when i use some clips for training and the other clips (coming from the same person) for validation, the result will be very high (but this does not mean i get a good performance).

**Comment by yujmo** [comment]
兄弟，你有这个数据集吗？能给我发一份吗？我的邮箱是yujmo@tongji.edu.cn

**Comment by YangYangTaoTao** [comment]
这个数据集很大164GB，邮箱应该搞不定的

**Comment by yujmo** [comment]
可以百度网盘共享一下吗。这个数据集也太大了吧

**Comment by YangYangTaoTao** [comment]
开源那我压缩一波之后发你邮箱吧

**Comment by yujmo** [comment]
邮箱不一定能装得下。 学校的邮箱只有15G的空间。   要是您方便的话，发我qq邮箱 yujmo@qq.com。QQ邮箱感觉应该可以的

**Comment by YangYangTaoTao** [comment]
我发你网盘链接

**Comment by yujmo** [comment]
OK，太感谢了。

**Comment by ligaoqi2** [comment]
> 我发你网盘链接

大佬，可以发我一份网盘链接吗？

**Comment by LwkBuaa** [comment]
可以发我一份网盘链接吗？我的邮箱是2933841829@qq.com

**Comment by seguneun2** [comment]
Hello, can I get a link to the dataset?
My email is smd3133dms@gmail.com .
Please.

**Comment by 1210571578** [comment]
> OK，兄弟，能不能发我一份，感谢

**Comment by 1210571578** [comment]
> 我发你网盘链接
您好，可以发一份数据集吗，非常感谢

**Comment by 1210571578** [comment]
打扰您一下，作者，可以方便给一下数据集吗，我的邮箱是1210571578@qq.com

**Comment by Mrhahaa** [comment]
@YangYangTaoTao 哥，我目前是一名计算机视觉学生，目前在试着做作者这个模型，请问这个数据集可以发给我一份吗？先谢谢了
我的邮箱：1511664995@qq.com，或者可以发我网盘链接，麻烦您了！
